examples/basic/main.tf

```diff
@@ -132,4 +132,8 @@ module "eks" {
   map_users_count                      = "${var.map_users_count}"
   map_accounts                         = "${var.map_accounts}"
   map_accounts_count                   = "${var.map_accounts_count}"
+
+  workers_assume_role_policy_principals_aws_identifiers = [
```

Comment:
While it's good to have examples to show things are done, the `examples/basic` directory is supposed to be very simple, and I don't think this item falls into that category 🙂

Reply:
I'm concerning... There isn't proper example to handle issue about IAM. But i think it's not good to create new example for only this variable.

Reply:
> i think it's not good to create new example for only this variable.

Not here. Please remove. Like I said, this is supposed to be the most basic example to get a cluster running. If you are including some random and invalid ARN then it's just confusing.